src/main/resources/org/cbioportal/persistence/mybatisclickhouse/StudyViewMapper.xml

```diff
@@ -37,8 +37,7 @@
                 <include refid="mutationAlterationFilter"/>
             </if>
         </where>
-        GROUP BY hugo_gene_symbol
-        ORDER BY totalCount DESC;
```

Comment:
Did you mean to lose the ORDER BY?

Reply:
Didn't think it was necessary since the frontend does its own filtering